fri_site/models.py

```diff
@@ -1,17 +1,32 @@
 from django.db import models
 from django.utils import timezone
+from model_utils.fields import StatusField
+from model_utils import Choices
 import datetime
 
 
-class Request(models.Model):
-    update_id = models.IntegerField(null=True)
-    request = models.TextField()
+class User(models.Model):
+    user_id = models.IntegerField(unique=True, primary_key=True)
+    first_name = models.CharField(max_length=64)
+    last_name = models.CharField(max_length=64, null=True)
+    user_name = models.CharField(max_length=64, null=True)
```

Comment:
probably this one is not null, other can be null because of hidden, but better to check on telegram docs